Sources/TokamakStaticHTML/Modifiers/Effects/OpacityEffect.swift

```diff
@@ -0,0 +1,24 @@
+// Copyright 2021 Tokamak contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//  Created by Carson Katri on 1/20/21.
+//
+
+import TokamakCore
+
+extension _OpacityEffect: DOMViewModifier {
+  public var attributes: [HTMLAttribute: String] {
+    ["style": "opacity: \(opacity)"]
```

Comment:
`Circle().frame(width: 100, height: 100, alignment: .center).opacity(0.5)`will render the `style` improperly:

`<div style="opacity: 0.5width: 100.0px;`...

```suggestion
    ["style": "opacity: \(opacity); "]
```

Reply:
Great catch, thanks!